client/post-editor/media-modal/image-editor/image-editor-canvas.jsx

```diff
@@ -127,6 +134,41 @@ const MediaModalImageEditorCanvas = React.createClass( {
 		MediaUtils.canvasToBlob( newCanvas, callback, this.props.mimeType, 1 );
 	},
 
+	/**
+	 * Compare initial `transform` properties with the given `transform` object
+	 * gotten from `props` component object and computes if the image has been transformed.
+	 *
+	 * @return {Boolean} `true` ist the image has changed
+	 */
+	hasTransformed() {
+		return JSON.stringify( defaults.transform ) !== JSON.stringify( this.props.transform );
```

Comment:
Hmm, I think https://lodash.com/docs/4.15.0#isEqual is probably a safer bet. The current example may need to sort it's keys.